script.js

```diff
@@ -0,0 +1,34 @@
+const header = document.querySelector('header');
+const title = document.getElementById('logo');
+const button = document.querySelector('.menu-bars');
+
+// const main = document.querySelector("main");
+
+const navbar = document.querySelector('nav');
+const button2 = document.createElement('button');
+button2.innerHTML = '<i class="fas fa-times fa-2x"></i>';
+
+const link = document.querySelector('.nav-link');
+const link2 = document.querySelector('.link2');
+const link3 = document.querySelector('.link3');
+
+function showNav() {
+  title.remove();
+  button.remove();
+  navbar.classList.remove('none');
+  header.appendChild(navbar);
+  header.appendChild(button2);
+}
+
+function showMenu() {
+  button2.remove();
+  navbar.classList.add('none');
+  header.appendChild(title);
+  header.append(button);
+}
```

Comment:
- [ ]REQUIRED: Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```